### ISSUE TITLE
Fixation-off sensitivity

### ISSUE THREAD

**Issue by DLewisSmith** [obophenotype/human-phenotype-ontology#6796]
**Preferred term label:**
Fixation-off sensitivity

**Synonyms**
Fixation-off epileptiform discharges

**Definition (free text, please give PubMed ID)**
The term "fixation off sensitivity" (FOS) was proposed to describe seizure and electroencephalography changes evoked by the suppression of central vision and fixation.

Comment:
The FOS EEG pattern usually consists of spikes/polyspikes and waves localized in occipital regions (bilateral or unilateral) or generalized discharges. It is usually inhibited by intermittent photic stimulation (IPS).

PMID: 24099056
**Parent term (use hpo.jax.org/app)**
Interictal epileptiform activity HP:0011182

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:139431

**Your nano-attribution (ORCID)**
/0000-0002-1735-8178

**Comment by pnrobinson** [comment]
@DLewisSmith 
We have revised the definition for non-specialists, could you check for accuracy please?

Fixation off sensitivity (FOS) is characterized by posterior or generalized epileptiform discharges that consistently occur with conditions that eliminate central vision, such as closed eyes, complete darkness, modified Ganzfeld stimulation (using a large white surface without visual cues), and Frenzel lenses. The FOS electroencephalogram (EEG) pattern usually consists of spikes/polyspikes and waves localized in occipital regions (bilateral or unilateral) or generalized discharges. It is usually inhibited by intermittent photic stimulation (e.g., flashing light or patterns used as stimuli during the recording of the EEG). 
PMID:22925838
PMID:24099056

Comment:
FOS occurs mainly in children who have idiopathic partial epilepsies with occipital paroxysms such as Gastaut syndrome and, less frequently, in Panayiotopoulos syndrome. It is also described in patients with eyelid myoclonia with absences and atypical features (i.e., cognitive impairment, absence status epilepticus) and in idiopathic generalized epilepsy with photosensitivity. In addition, FOS has been documented in nonepileptic patients with posterior paroxysms. Lastly, this pattern has, on rare occasions, been observed in symptomatic or cryptogenic epilepsies. In all the aforementioned cases, the FOS phenomenon is typical of infancy or childhood, with reports in adult epileptic patients being an exception.

**Comment by DLewisSmith** [comment]
Thank you Peter, I like it.


I would alter the final sentance thus:

Fixation off sensitivity (FOS) is characterized by posterior or generalized epileptiform discharges that consistently occur with conditions that eliminate central vision, such as closed eyes, complete darkness, modified Ganzfeld stimulation (using a large white surface without visual cues), and Frenzel lenses. The FOS electroencephalogram (EEG) pattern usually consists of spikes/polyspikes and waves localized in occipital regions (bilateral or unilateral) or generalized discharges. It is usually inhibited by fixation of gaze, observing patterns, or intermittent photic stimulation (e.g., flashing light or patterns used as stimuli during the recording of the EEG).

In the comment, I would alter the wording of the penultimate sentence as we no longer use 'symptomatic' or 'cryptogenic' and shorten the comment:

FOS occurs mainly in children who have non-acquired focal epilepsies with occipital epileptiform discharges and in people with generalized epilepsies with photosensitivity such as eyelid myoclonia with absences. It has been observed in people with epilepsies due to structural brain lesions and people without epilepsy who have posterior epileptiform discharges.

**Comment by pnrobinson** [comment]
added term
